lib/SPIRV/SPIRVReader.cpp

```diff
@@ -987,6 +995,45 @@ void SPIRVToLLVM::setLLVMLoopMetadata(const LoopInstType *LM,
   }
   if (LC & LoopControlNoFusionINTELMask)
     Metadata.push_back(getMetadataFromName("llvm.loop.fusion.disable"));
+  if (LC & spv::internal::LoopControlLoopCountINTELMask) {
+    // LoopCountINTELMask parameters are int64 and each parameter is stored
+    // as 2 SPIRVWords (int32)
+    uint64_t LoopCountMin =
+        static_cast<uint64_t>(LoopControlParameters[NumParam++]);
+    LoopCountMin =
+        LoopCountMin | static_cast<uint64_t>(LoopControlParameters[NumParam++])
+                           << 32;
```

Comment:
```suggestion
    LoopCountMin |=
        static_cast<uint64_t>(LoopControlParameters[NumParam++]) << 32;
```

Reply:
Changed it in this commit: [a7a3741](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1062/commits/a7a3741d55a23ee5bb5298c087fc5eaf778edda5)